backend/app/store/service/restricted_words.go

```diff
@@ -0,0 +1,95 @@
+package service
+
+import (
+	"fmt"
+	"strings"
+	"unicode"
+)
+
+type Lister interface {
+	List(siteID string) (restricted []string, err error)
+}
+
+type StaticLister struct {
```

Comment:
I don't think static lister should have map here. All other static implementations of similar things we have just ignore site and have a shared list